### DISCUSSION TITLE
Will any upcoming release will support Django 3.2

### DISCUSSION THREAD

**Discussion by rehan252** [pyinstaller/pyinstaller#6234] [Help]
I am currently working on a project build on Python 3.9 and Django 3.2 and Django-Channels 3.1 . The requirements came to convert this app into an executable single app. The main goal is to make my application downloadable to use as an offline desktop application. 
But current PyInstaller only supports Django=2.1. So, is there any workaround for Django=3.2 ?

**Comment by rokm** [comment]
> Will any upcoming release will support Django 3.2

As far as I know, nobody is working towards that. I'm not even sure in what state our Django support (official or unofficial) is. But I imagine that at the very least, you'll need to work around the problems with `runserver` (i.e., by using `--noreload`, as per #5294).
